devicetypes/smartthings/zigbee-dimmer-power.src/zigbee-dimmer-power.groovy

```diff
@@ -0,0 +1,112 @@
+/**
+ *  Copyright 2015 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+metadata {
+    definition (name: "ZigBee Dimmer Power", namespace: "smartthings", author: "SmartThings") {
+        capability "Actuator"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Power Meter"
+        capability "Sensor"
+        capability "Switch"
+        capability "Switch Level"
+
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B04"
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702"
+    }
+
+    // simulator metadata
+    simulator {
+        // status messages
+        status "on": "on/off: 1"
+        status "off": "on/off: 0"
+
+        // reply messages
+        reply "zcl on-off on": "on/off: 1"
```

Comment:
these are simulated replies for the IDE. i agree that we need to revisit the code for simulator